EventsExpress.Core/Services/InventoryService.cs

```diff
@@ -34,6 +34,7 @@ public async Task<OperationResult> AddInventar(Guid eventId, InventoryDTO invent
             {
                 var entity = _mapper.Map<InventoryDTO, Inventory>(inventoryDTO);
                 entity.EventId = eventId;
+                entity.UnitOfMeasuring = null;
```

Comment:
Why do we have to explicitly set it to null? Is it not null when coming from mapper?

Reply:
Now it null when coming from mapper